core/api/src/graphql/error.ts

```diff
@@ -381,9 +381,9 @@ export class InvalidPhoneForOnboardingError extends CustomGraphQLError {
 export class UnauthorizedIPMetadataCountryError extends CustomGraphQLError {
   constructor(errData: CustomGraphQLErrorData) {
     super({
-      message: "Country not not authorized for rewards.",
+      message: "Country not not authorized for quizzes.",
       forwardToClient: true,
-      code: "UNAUTHORIZED_COUNTRY_IP_FOR_REWARD",
+      code: "UNAUTHORIZED_COUNTRY_IP_FOR_QUIZZES",
```

Comment:
does this break mobile app error handling?

Reply:
this string doesn't exist on mobile: https://github.com/search?q=repo%3AGaloyMoney%2Fgaloy-mobile%20UNAUTHORIZED_COUNTRY_IP_FOR_REWARD&type=code

Reply:
same for the 2 below